src/app/composer/qgscomposer.cpp

```diff
@@ -2136,6 +2136,7 @@ void QgsComposer::exportCompositionAsImage( QgsComposer::OutputMode mode )
     QFileDialog dlg( this, tr( "Export atlas to directory" ) );
     dlg.setFileMode( QFileDialog::Directory );
     dlg.setOption( QFileDialog::ShowDirsOnly, true );
+    dlg.setOption( QFileDialog::DontUseNativeDialog, true );
```

Comment:
it would make sense add a comment in ase other try to change QFileDialog::DontUseNativeDialog and having side effect in other parts of the code? giving a renderence of the code that this flag affect, could avoid wrong modifications int he future.